src/libmongoc/tests/mock_server/mock-server.c

```diff
@@ -1103,22 +1081,23 @@ mock_server_receives_legacy_hello (mock_server_t *server,
  *       request is not a hello command.
  *
  * Side effects:
- *       Logs if the current request is not a hello command.
+ *       Logs if the current request is a hello command using OP_QUERY.
  *
  *--------------------------------------------------------------------------
  */
 
 request_t *
 mock_server_receives_hello (mock_server_t *server)
 {
+   /* Note that mock_server_receives_command() expects JSON documents, not BSON
+   documents, unlike _mock_server_receives_msg(): */
```

Comment:
This comment is misleading, please remove. `mock_server_receives_command` expects a null-terminated byte string corresponding to a single document, whereas `mock_server_receives_msg` expects a series of `bson_t` objects corresponding to one or more documents. The content of the documents are specified in exactly the same way (no distinction between "JSON" and "BSON" documents between these two functions).